client/packages/common/src/intl/locales/en/common.json

```diff
@@ -72,7 +72,7 @@
   "label.actions": "Actions",
   "label.address": "Address",
   "label.allocated": "Allocated",
-  "label.already-issued": "Issued",
+  "label.unissued": "Unissued",
```

Comment:
ok - so this has confused me! It's not clear what the difference is between `Unissued` and `Remaining`. and trying to figure out what the numbers mean by working with a requisition isn't helping me.

I remember Richard asking for changes to the column names and functionality.. but cannot find record of it. Only these two issues:
https://github.com/openmsupply/openmsupply-client/issues/1036
https://github.com/openmsupply/remote-server/issues/1017

I'm thinking to remove the 'unissued' column and just show 'Requested', 'Supply', 'Remaining'
with an updated to the descriptions
`Requested: The quantity requested by the customer`
`Supply: The quantity that will be supplied by this store`
`Remaining: The quantity remaining to be supplied, which is the Supply quantity less the quantity of this item on any linked Outbound Shipments`

Another suggestion 
Change `label.supply-quantity` to `Supply` - since the term `Quantity` is pretty unhelpful! Could apply to any of the numeric fields.

Reply:
Ah yes, unissued is remaining - amount in outbound invoice ~. Will change (: